homeassistant/components/stiebel_eltron/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""The component for STIEBEL ELTRON heat pumps with ISGWeb Modbus module."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.modbus import (
+    CONF_HUB, DEFAULT_HUB, DOMAIN as MODBUS_DOMAIN)
+from homeassistant.const import CONF_NAME, DEVICE_DEFAULT_NAME
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pystiebeleltron==0.0.1.dev2']
+DEPENDENCIES = ['modbus']
```

Comment:
We have removed `REQUIREMENTS` and `DEPENDENCIES` from all existing modules now. We're only relying on the manifest.json file moving forward. Please remove these here.

Reply:
Done, and adjusted the import in climate.py to run it as a custom_components.